README.md

```diff
@@ -22,24 +22,29 @@ Add the preset to your `jest.config.js` or `jest` field in `package.json`:
 }
 ```
 
+### Adjust setup globally
+
 Optionally, you can avoid specifying `plugins` in every schema by defining your own setup file to configure the `testRule`/`testRuleConfigs` functions.
 This is useful if you have many tests. There are two additional steps to do this:
 
 1. Create `jest.setup.js` in the root of your project. Provide `plugins` option to `getTestRule`/`getTestRuleConfigs`:
 
    ```js
    const { getTestRule, getTestRuleConfigs } = require("jest-preset-stylelint");
+   const myPlugin = require("./my-plugin.js");
+
+   const plugins = [myPlugin];
 
-   global.testRule = getTestRule({ plugins: ["./"] });
-   global.testRuleConfigs = getTestRuleConfigs({ plugins: ["./"] });
+   global.testRule = getTestRule({ plugins });
+   global.testRuleConfigs = getTestRuleConfigs({ plugins });
    ```
```

Comment:
[note] I think keeping CJS for `jest.setup.js` is better because Jest's ESM support is still experimental.

Reply:
I misunderstood. `jest.setup.js` in ESM works, too. Fixed with ec31a1b.